curriculum/challenges/english/08-coding-interview-prep/rosetta-code/run-length-encoding.md

```diff
@@ -0,0 +1,117 @@
+---
+id: 5eb257ba998e23057d209754
+title: Run-length Encoding
+challengeType: 5
+forumTopicId: 387031
+---
+
+## Description
+<section id='description'>
+Given a string containing uppercase characters (A-Z), compress repeated 'runs' of the same character by storing the length of that run, and provide a function to reverse the compression.
+
+Example:<br>
+Original: `ABBBCDDDD`<br>
+Encoded: `[[1, 'A'], [3, 'B'], [1, 'C'], [4, 'D']]`
```

Comment:
```suggestion
Original: `ABBBAAZCDDDD`<br>
Encoded: `[[1, 'A'], [3, 'B'], [2, 'A'], [1, 'Z'], [1, 'C'], [4, 'D']]`
```
I'm suggesting this change to the example so that the user does not think that everything needs to be in alphabetical order. They can figure it out using the tests, but I'd rather add extra clarity.